front/gustavo/tarefa4JS/main.js

```diff
@@ -0,0 +1,127 @@
+
+// se a classe estiver no elemento, o toggle remove e retorna false. Se não existir, ele adiciona e retorna true
+let telafinal = document.getElementsByClassName('indetificaessamerda')[0]; 
+telafinal.classList.add('screenbuy');
+
+
+let desert = false;let chiken = false; let drink = false; var foodToBy = []; var foodToByP = []; var drinkToBy = []; var drinkToByP = [] ;var desertToBy = []; var desertToByP = []
```

Comment:
Aconselho não usar `var`, apenas `let` e `const`, conforme a necessidade

![image](https://github.com/einsteinfloripa/dev-weekend/assets/97882083/9640b350-fd9f-460d-80de-a7b86bad904e)

a um bom tempo já estamos no ES6

![image](https://github.com/einsteinfloripa/dev-weekend/assets/97882083/a1fe1e9c-0e51-4f96-8d27-949d646f9a51)

Também não é boa prática geral de organização de código fazer isso, essa "concatenação" de variáveis, é melhor dividir por linha

```suggestion
let desert = false;
let chiken = false; 
let drink = false; 
var foodToBy = []; 
var foodToByP = []; 
var drinkToBy = []; 
var drinkToByP = []; 
var desertToBy = []; 
var desertToByP = []
```

Reply:
Ok